include/regs.h

```diff
@@ -58,15 +58,26 @@
 #define R_FB_FILTER_ENV_COLOR(c)                 SREG(85 + (c))
 #define R_ENABLE_FB_FILTER                       SREG(88)
 #define R_PAUSE_MENU_MODE                        SREG(94)
+#define R_CAM_XZ_UPDATE_RATE                     OREG(2)
+#define R_CAM_Y_OFFSET_UPDATE_RATE               OREG(3)
+#define R_CAM_FOV_UPDATE_RATE                    OREG(4)
 #define R_CAM_MAX_PITCH                          OREG(5)
+#define R_CAM_R_UPDATE_RATE_INV                  OREG(6)
 #define R_CAM_DEFAULT_PITCH_UPDATE_RATE_INV      OREG(7)
+#define R_CAM_SLOPE_Y_ADJ_ADJ_AMOUNT             OREG(9)
 #define R_CAM_PITCH_FLOOR_CHECK_NEAR_DIST_FAC    OREG(17)
 #define R_CAM_PITCH_FLOOR_CHECK_FAR_DIST_FAC     OREG(18)
 #define R_CAM_PITCH_FLOOR_CHECK_OFFSET_Y_FAC     OREG(19)
 #define R_CAM_PITCH_FLOOR_CHECK_NEAR_WEIGHT      OREG(20)
 #define R_CAM_DEFAULT_ANIM_TIME                  OREG(23)
+#define R_CAM_UPDATE_RATE_STEP_SCALE_XZ          OREG(25) // name not ideal
+#define R_CAM_UPDATE_RATE_STEP_SCALE_Y           OREG(26) // name not ideal
+#define R_CAM_JUMP1_EYE_Y_STEP_SCALE             OREG(31)
```

Comment:
Is `stepScale` synonomous with a lerp param ranging from 0-1?
`eyeNext->y += (newEye.y - eyeNext->y) * CAM_DATA_SCALED(R_CAM_JUMP1_EYE_Y_STEP_SCALE);`
is a lerp with `newEye` as the target

Reply:
Yes
`Camera_LERPCeilF`'s `stepScale` at least, which is used like `cur + (target - cur) * stepScale`